Pending status

grahammcculloch opened the pull request:
**Reset pending status on support thread activity**
Change-type: minor
Signed-off-by: Graham McCulloch <graham@balena.io>
***
**ui: Add Pending Status toggle button**
Displays the pending status of a support thread (dark blue background if pending) and lets the user toggle the pending status.

Change-type: minor
Signed-off-by: Graham McCulloch <graham@balena.io>
***
**Add isPending boolean field to support-thread and sales-thread**
Change-type: minor
Signed-off-by: Graham McCulloch <graham@balena.io>
***

![Pending Status](https://user-images.githubusercontent.com/2925657/96109771-37c96580-0f09-11eb-846a-0dd8ca6e8aee.gif)

grahammcculloch committed:
Add isPending boolean field to support-thread and sales-thread

Change-type: minor
Signed-off-by: Graham McCulloch <graham@balena.io>

grahammcculloch committed:
ui: Add Pending Status toggle button

Displays the pending status of a support thread (dark blue background if pending) and lets the user toggle the pending status.

Change-type: minor
Signed-off-by: Graham McCulloch <graham@balena.io>

grahammcculloch committed:
Reset pending status on support thread activity

Change-type: minor
Signed-off-by: Graham McCulloch <graham@balena.io>

LucianBuzzo reviewed (dismissed):
Clean implementation, I like it!

grahammcculloch committed:
Set ESR Version

This is a temporary solution to get CI to work on the feature branch.

Change-type: patch
Signed-off-by: Graham McCulloch <graham@balena.io>
